scripts/load_testing/client_config.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+from math import ceil
+import os
+
+
+INSTANCES = 3
+PREFIX = 'loadgen'
+RPCPERF_THREADS = 1
+RPCPERF_CONNS = 100
+RPCPERF_RATE = 10000
+RPCPERF_GET_RATIO = 0.9
+RPCPERF_SET_RATIO = 1 - RPCPERF_GET_RATIO
+PELIKAN_SLAB_MEM = 4294967296
+PELIKAN_ITEM_OVERHEAD = 48
+KSIZE = 32
+VSIZE = 32
+PELIKAN_SERVER_PORT = 12300
+PELIKAN_SERVER_IP = '10.25.2.44'
+RPCPERF_BINARY = '/root/Twitter/rpc-perf/target/release/rpc-perf'
+
+
+def generate_config(rate, connections, vsize, slab_mem, threads):
+# create rpcperf.toml
+  rate_get = int(rate * RPCPERF_GET_RATIO)
+  rate_set = int(rate * RPCPERF_SET_RATIO)
+  nkey = int(ceil(1.0 * slab_mem / (vsize + KSIZE + PELIKAN_ITEM_OVERHEAD)))
+  conn_per_thread = connections / threads
+
+  config_str = """\
+[general]
+threads = {threads}
+tcp-nodelay = true
+connections = {connections}
+windows = 600
+duration = 60
+request-timeout = 200
+connect-timeout = 50
```

Comment:
50ms connect timeout? Seems aggressive compared to the request timeout.

Reply:
200/200?